src/bundles/SellerRegistration/redux/modules/reducer-signup.js

```diff
@@ -56,9 +53,6 @@ export default combineReducers({
     },
```

Comment:
You will need to add the reference fields down here too, this is important in the event of a dispatched reset.